public/css/custom.css

```diff
@@ -71,7 +71,7 @@ body {
   font-size: var(--bs-body-font-size);
   font-weight: var(--bs-body-font-weight);
   line-height: var(--bs-body-line-height);
-  color: var(--bs-body-color);
+  color: white;;
```

Comment:
Line 53 and 74, this file should not be edited directly, it is automatically generated and will be overwritten. I'll address this in a fix after it is merged.